src/entities/communitySbt.ts

```diff
@@ -106,7 +106,7 @@ class SBT {
         if(!rootEntity) {
             throw new Error('No root found')
         }
-        const metadataUri = `ipfs:${String.fromCharCode(47)}${String.fromCharCode(47)}${rootEntity.baseMetadataUri}${String.fromCharCode(47)}${badgeType}.json`;
+        const metadataUri = `${rootEntity.baseMetadataUri}${badgeType}.json`;
```

Comment:
feels like you need to refactor it as util to avoid the code repetition and keep logic in one place. I am approving this for now, but can you add a task for yourself to fix this in next PR perhaps?